background/background.js

```diff
@@ -1,7 +1,7 @@
 const backgroundsMapping = {
 	"200": "./assets/thunders/thunders.jpg", //Thunderstorm
 	"300": "./assets/rainy/Drizzle.jpg", //Drizzle
-	"500": "./assets/rainy/Image9-kopia.jpg", //Rain
+	"500": "./assets/rainy/Image9.jpg", //Rain
```

Comment:
You could rename  those files to say what they represent.
Ex. instead of Image9.jpg, name it rain.jpg

Reply:
You're right, I changed other filenames too according to their meaning.